blitzloop/play.py

```diff
@@ -43,6 +46,9 @@
     display = graphics.Display(1280, 720, fullscreen, None)
 print(display.width, display.height)
 
+audio = AudioEngine()
```

Comment:
Where did this come from? I deliberately removed _audio (aka mic echo) from play.py when I switched to mpv, because it's annoying more often than not (mic feedback on laptops), and play.py has no controls for it. If this is going to stay it needs a commandline argument to enable it.